fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/service/message/domain/MessageCreatedAt.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.event.external.service.message.domain;
+
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import lombok.Getter;
+
+@Getter
+public class MessageCreatedAt {
+
+    private OffsetDateTime createdAt;
+
+    public MessageCreatedAt(OffsetDateTime createdAt) {
+        this.createdAt = Objects.requireNonNull(createdAt, "createdAt cannot be null");
```

Comment:
Please extract the string into a constant.

Reply:
Same as the other.